package.json

```diff
@@ -38,7 +38,11 @@
     "yosay": "^2.0.1"
   },
   "jest": {
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "generators/app/index.js"
```

Comment:
I suggest use `collectPathIgnorePatterns` to simply ignore the `__test__` and `template`. Overall the code might go beyond a single `generators/app/index.js` and it is not easy to notice that the coverage is not collected for other js files. In this case they will NOT appear on the coverage report.